pjsip/src/pjsua-lib/pjsua_call.c

```diff
@@ -961,8 +961,13 @@ PJ_DEF(pj_status_t) pjsua_call_make_call(pjsua_acc_id acc_id,
 
     if( opt->custom_call_id.slen > 0 ){
         pj_strdup(dlg->pool, &dlg->call_id->id, &opt->custom_call_id);
-        PJ_LOG(4,(THIS_FILE, "Set user defined Call-ID (%.*s)", (int)dlg->call_id->id.slen, dlg->call_id->id.ptr  ));
-    }
+        PJ_LOG(4,(THIS_FILE, "Set user defined "
+                             "Call-ID (%.*s)",
+                  (int)dlg->call_id->id.slen, dlg->call_id->id.ptr  ));
+        pj_bzero(&call->opt.custom_call_id,
+                  sizeof(call->opt.custom_call_id));
+        call->opt.custom_call_id.slen=0;
```

Comment:
This line seems redundant as the `bzero()` above should already set `slen` to zero.